### ISSUE TITLE
error on initial python setup_casapy_pip.py

### ISSUE THREAD

**Issue by brisbind** [radio-astro-tools/casa-python#7]
I have casapy 4.2.0. When I try the initial python setup_casapy_pip.py

```
(yt-x86_64)[dbrisbin@mach Downloads]$ python setup_casapy_pip.py
Traceback (most recent call last):
  File "setup_casapy_pip.py", line 227, in <module>
    python_version = get_python_version_linux()
  File "setup_casapy_pip.py", line 53, in get_python_version_linux
    casapy_path = get_casapy_path()
  File "setup_casapy_pip.py", line 38, in get_casapy_path
    return os.path.realpath(find_executable('casapy'))
  File "/export/data_1/dbrisbin/Downloads/yt-x86_64/lib/python2.7/posixpath.py", line 367, in realpath
    path, ok = _joinrealpath('', filename, {})
  File "/export/data_1/dbrisbin/Downloads/yt-x86_64/lib/python2.7/posixpath.py", line 373, in _joinrealpath
    if isabs(rest):
  File "/export/data_1/dbrisbin/Downloads/yt-x86_64/lib/python2.7/posixpath.py", line 61, in isabs
    return s.startswith('/')
AttributeError: 'NoneType' object has no attribute 'startswith'
```

**Comment by astro313** [comment]
Is there a fix to this, I am getting identical error, see below:

I successfully installed casa-python on my own machine without any issue (CASA 4.2.2 and python 2.7.9), whereas currently I am getting the following error when I ran `python setup_casapy_pip.py` on another machine with CASA 4.3.1 and python 2.7.9:
```
Traceback (most recent call last):
  File "setup_casapy_pip.py", line 250, in <module>
    python_version = get_python_version_linux()
  File "setup_casapy_pip.py", line 82, in get_python_version_linux
    version,casapy_parent_path = get_python_path_linux()
  File "setup_casapy_pip.py", line 78, in get_python_path_linux
    raise ValueError("Could not determine Python version")
ValueError: Could not determine Python version
```

`which python` returns ~/anaconda/bin/python
`python -V` returns Python 2.7.9 :: Anaconda 2.1.0 (64-bit)
`which casapy` returns ~/CASA-4.3.1/casa-release-4.3.1-el6/casapy

**Comment by keflavich** [comment]
@astro313 - this should now be fixed by #14.  Could you test and let us know if it worked?

**Comment by astro313** [comment]
Yes, it worked for me! CASA 4.3.1 on Linux
